bb.edn

```diff
@@ -31,6 +31,16 @@
   dev:app-watch
   logseq.tasks.dev.mobile/app-watch
 
+  dev:build-publishing
+  logseq.tasks.dev/build-publishing
+
+  dev:publishing
+  {:depends [dev:build-publishing]
+   :doc "Build publishing spa app given graph and output dirs"
+   :task (apply shell {:dir "scripts"}
+           "yarn -s nbb-logseq -cp src -m logseq.tasks.dev.publishing"
```

Comment:
Got this error: Command "nbb-logseq" not found.`npm install @logseq/nbb-logseq -g` has been executed and nbb-logseq can be invoked.

Reply:
Sorry forgot to include one time setup. If you `cd scripts && yarn install && cd -`, does the bb command work?

Reply:
It's working now! 🚢